demo/scripts/run-testnet.sh

```diff
@@ -32,6 +32,7 @@ do
     --client-connect="172.77.5.$(($N+$i)):1339" \
     --service-listen="172.77.5.$i:80" \
     --sync-limit=1000 \
+    --store = true \
```

Comment:
I think those spaces around `=` don't work with cobra. Also as a boolean parameter you could just write `--store`